data/test_endpoints/event_brokers/pika_endpoint.yml

```diff
@@ -1,6 +1,10 @@
 event_broker:
+  type: pika
   url: localhost
   username: username
   password: password
-  queue: queue
-  type: pika
+  queue: queue  # to publish to a single queue
+#  alternatively, you can publish to multiple queues with:
+#  queues:
+#    - queue-1
+#    - queue-2
```

Comment:
Would it be less confusing if we just changed the parameter to `queues` (with backwards compatiblity for `queue`) and handled both a string and a list for `queues` (or made it have to be a list)?

Reply:
this is essentially what happens, except that we're not showing a FutureWarning. I'm not sure about deprecating `queue` and I think a single queue will remain the most common case

Reply:
I'm sure it will, just wondering if it makes more sense to only have one param. I don't really feel strongly about it though, just thought i'd bring it up

Reply:
I'd agree though, would be a bit less confusing to recommend and document `queues` only, then deprecate `queue`